pkg/ottl/ottlfuncs/README.md

```diff
@@ -1358,6 +1359,21 @@ Examples:
 
 - `Year(Now())`
 
+### Scale
+
+`Scale(value, factor)`
+
+The `Scale` function returns the original `value`, multiplied by the `factor`.
+The supported data types are:
+
+- `int`
+- `double`
+- `metric` - Supported metric types are `Gauge`, `Sum`, and `Histogram`
```

Comment:
We should have something here to call out the `data_points` key of the `metric` context. We should also probably have an example that calls something like `set(data_points, Scale(data_points, 0.1))` to make it clear how this is used.

Reply:
good point, i have added a more detailed description